<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="centralwidget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>289</width>
      <height>143</height>
     </rect>
    </property>
    <widget class="QComboBox" name="comboBoxA">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelSum">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCarry">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>100</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelInputA">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Input A</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelInputB">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Input B</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSumText">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Sum</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelCarryText">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Carry</string>
     </property>
    </widget>
   </widget>
   <widget class="QMenuBar" name="menubar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QStatusBar" name="statusbar_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>3</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
